include/os/linux/kernel/linux/simd_x86.h

```diff
@@ -135,7 +135,18 @@
  */
 #if defined(HAVE_KERNEL_FPU_INTERNAL)
 
+/*
+ * Unmask the kernel SLAB interface, re-define the SPL wrappers further below.
+ */
+#if defined(kmem_cache_create)
+#define reset_spl_kmem_cache_definitions
```

Comment:
I already fixed the whitespace nit locally, but I wanted to wait for the test runners to finish (and any other feedback) before force-pushing again.